Simplified/Accounts.swift

```diff
@@ -159,7 +159,7 @@ final class Account:NSObject
   {
     defaults = UserDefaults.standard
     
-    name = json["name"] as? String
+    name = json["name"] as! String
```

Comment:
I feel like we would want to crash in dev but log in production. If we have a mechanism for doing so I'd say we should use it instead of a force unwrap.

Reply:
The crash will occur on launch if the force unwrap fails. Unless we release an application we've literally never tried to run, there's no way we'll have this issue in production.

I don't think it's worth the complexity of multiple code paths which depend on compile-time settings. They'd introduce an additional case that would be untestable in debug mode and therefore would actually be _more_ likely to result in a bug in production. Just my opinion, of course!